packages/config-eslint/.eslintrc.js

```diff
@@ -0,0 +1,72 @@
+module.exports = {
+  root: true,
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+  },
+  plugins: [
+    '@typescript-eslint',
+    'react',
+    '@next/eslint-plugin-next',
+    'prefer-arrow-functions',
+    'import',
+    'jsx-a11y',
+  ],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@next/next/recommended',
+    'plugin:jsx-a11y/recommended',
+  ],
+  rules: {
+    'react/jsx-sort-props': 'error',
+    'import/order': [
+      'error',
+      {
+        alphabetize: {
+          caseInsensitive: true,
+          order: 'asc',
+        },
+        groups: [
+          'builtin',
+          'external',
+          'internal',
+          'parent',
+          'sibling',
+          'index',
+          'object',
+          'type',
+        ],
+        'newlines-between': 'always',
+        pathGroupsExcludedImportTypes: ['builtin'],
+      },
+    ],
+    'react/function-component-definition': [
+      'error',
+      { namedComponents: 'arrow-function' },
+    ],
+    'import/newline-after-import': 'error',
+  },
+  overrides: [
+    {
+      //Biomeとかぶっているため、一旦無効化
```

Comment:
```suggestion
      // Biome と重複しているため、ESLint では無効にする。
```

などでどうでしょうか？ 「かぶっている」は口語的な気がしました。

Reply:
IMO、コメントは英語で残すのが僕は好きです。

Reply:
僕も英語派です
英語にします？

Reply:
それなら英語にしましょう！